config.template.js

```diff
@@ -1,5 +1,4 @@
 // copy this to your config file
-module.exports = {
```

Comment:
Why is this removed?

Reply:
by mistake. I'll fix it in my next pull request.

Reply:
Can you fix and push the branch again? If we merge it will break the master branch